hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson

```diff
@@ -14,19 +14,19 @@
       desc: '''Drive unsupported requests via TL interface and verify correctness of response
             / behavior. Below error cases are tested
             - TL-UL protocol error cases
-              - Unsupported opcode. e.g a_opcode isn't Get, PutPartialData or PutFullData
-              - Mask isn't all active if opcode = PutFullData
-              - Mask isn't in enabled lanes, e.g. a_address = 0x00, a_size = 0, a_mask = 'b0010
-              - Mask doesn't align with address, e.g. a_address = 0x01, a_mask = 'b0001
-              - Address and size aren't aligned, e.g. a_address = 0x01, a_size != 0
-              - Size is over 2.
+              - unsupported opcode. e.g a_opcode isn't Get, PutPartialData or PutFullData
+              - mask isn't all active if opcode = PutFullData
+              - mask isn't in enabled lanes, e.g. a_address = 0x00, a_size = 0, a_mask = 'b0010
+              - mask doesn't align with address, e.g. a_address = 0x01, a_mask = 'b0001
+              - address and size aren't aligned, e.g. a_address = 0x01, a_size != 0
+              - size is over 2
             - OpenTitan defined error cases
```

Comment:
Perhaps say "Error cases based on OpenTitan [TLUL spec](...):"; add link to TLUL spec.

Reply:
I think both categories are included in the tlul spec: https://docs.opentitan.org/hw/ip/tlul/doc/index.html#explicit-error-cases I believe we are just trying to distinguish if that is a common TLUL error case, or defined specifically by OpenTitan.